src/clarity-angular/data/tree-view/tree-view.module.ts

```diff
@@ -9,20 +9,20 @@ import { CommonModule } from "@angular/common";
 import { ClrIconModule } from "../../icon/icon.module";
 import { ClrFormsModule } from "../../forms/forms.module";
 import { FormsModule } from "@angular/forms";
-import { TREE_VIEW_DIRECTIVES } from "./index";
+import {NEW_TREE_VIEW_DIRECTIVES} from "./index";
 
 @NgModule({
     imports: [
         CommonModule,
         FormsModule, //[(ngModel)] usage in TreeView requires it.
```

Comment:
Are we using `ngModel` ourselves, in the view of our components? If not, we don't need to import `FormsModule`.